modules/n1ql/pages/n1ql-language-reference/nest.adoc

```diff
@@ -807,9 +804,9 @@ FROM _lhs-expression_
 
 There are three important changes in the index scan syntax example above:
 
-* `CREATE INDEX` on the `ON KEY` expression `route.airlineid` to access `route` documents using `airlineid` (which are produced on the left-hand side).
-* The `ON KEY route.airlineid FOR airline` enables N1QL to use the index `route.airlineid`.
-* Create any optional index, such as `route.airline` that can be used on `airline` (left-hand side).
+* `CREATE INDEX` on the `ON KEY` expression `rte.airlineid` to access `route` documents using `airlineid` (which are produced on the left-hand side).
```

Comment:
`rte` seems ugly abbreviation here, could the alias be named `route` to keep the example as it was before?

Reply:
In fact from other examples later, it looks like simply omitting the alias would let you refer to it as `route`

Reply:
The query actually says `ON KEY rte.airlineid`. Updated the explanation to match the query.